release/start-acs/action.yml

```diff
@@ -33,8 +33,16 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Docker login to quay.io
+      uses: docker/login-action@v3
+      with:
+        registry: quay.io
+        username: ${{ inputs.registry-username }}
+        password: ${{ inputs.registry-password }}
+
     - id: launch-central
       env:
+        ROX_PRODUCT_BRANDING: RHACS_BRANDING
```

Comment:
I think, we are missing a `$` somewhere here as this string is taken as flavor literaly

Reply:
I don't understand. "RHACS_BRANDING" is a string. It is used [here](https://github.com/stackrox/stackrox/blob/4b1004b8d07e8242d7b425766771411cbf44823b/Makefile#L62-L64) to determine the default location for the images ("DEFAULT_IMAGE_REGISTRY").

Reply:
Sorry, you are right. Your previous answer that the error is fixed addressed what I needed.